tests/scripts/run-xunit-perf.cmd

```diff
@@ -15,6 +15,7 @@ setlocal
   set TEST_CONFIG=Release
   set IS_SCENARIO_TEST=
   set USAGE_DISPLAYED=
+  set UPLOAD_TO_BENCHVIEW=0
```

Comment:
you could just make sure the variable is not defined: `set VAR=`, then check if it is defined (treat it as a Boolean): `if defined VAR`